scripts/install_deps.sh

```diff
@@ -93,8 +93,8 @@ case $(uname -s) in
             10.15)
                 echo "Installing solidity dependencies on macOS 10.15 Catalina."
                 ;;
-            11.0 | 11.1)
-                echo "Installing solidity dependencies on macOS 11.0 / 11.1 Big Sur."
+            11.0 | 11.1 | 11.2)
+                echo "Installing solidity dependencies on macOS 11.0 / 11.1 Big Sur/ 11.2 Big Sur."
```

Comment:
Can you put this into its own PR please?

Reply:
Done in PR #11231